### DISCUSSION TITLE
.NET MAUI 8.0.10 (Service Release 3)

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#21164] [Announcements]
This release is now available via [NuGet.org](https://www.nuget.org/packages/Microsoft.Maui.Controls/8.0.10) and is compatible with the latest .NET 8 workload. 

```
<ItemGroup>
  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.10" />
  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.10" />
</ItemGroup>
```

The focus of our service releases is resolving high-impact bugs for developers upgrading their applications from Xamarin to .NET 8.

- [Release notes](https://github.com/dotnet/maui/releases/tag/8.0.10)

**Comment by ziomek64** [comment]
https://github.com/dotnet/maui/issues/14471 
almost 1 year anniversary <3

**Comment by Kebechet** [comment]
I am still surprised that basic keyboard is still not workin properly
https://github.com/dotnet/maui/issues/19214#issuecomment-1997809647

**Comment by HuaFangYun** [comment]
I feel that the merging of issues submitted by the community is too slow, and I think if the fix is OK, it can speed up the merging of issues submitted by the community. In this way, community users may update and actively participate in and accelerate the development of Maui. Because I hear a lot of complaints about Maui problems and slow community consolidation. A lot of users are waiting for the project.
